src/test/java/com/exasol/MetadataReadingIT.java

```diff
@@ -144,4 +145,23 @@ void testProtectTableAfterRefresh() throws SQLException {
         refreshVirtualSchema(virtualSchema);
         assertRlsQueryWithUser("SELECT * FROM " + virtualSchema.getName() + ".T", user, table().row(true).matches());
     }
+
+    @Test
+    void testTableFilter() throws SQLException {
+        final Schema sourceSchema = createSchema("SCHEMA_FOR_TABLE_FILTER");
+        sourceSchema.createTable("T1", "C1_1", "BOOLEAN");
+        sourceSchema.createTable("T2", "C2_1", "BOOLEAN");
+        sourceSchema.createTable("T3", "C3_1", "BOOLEAN");
+        final VirtualSchema virtualSchema = createVirtualSchema(sourceSchema, Map.of("TABLE_FILTER", "T2, T3"));
+        final String sql = "/*snapshot execution*/"
```

Comment:
What does `snapshot execution` mean?

Reply:
This is a little trick to avoid creating table locks when scanning system tables. As long as you only read, you can tell transaction management that you are okay with looking at a snapshot.